src/core/Table/TablePaginator.tsx

```diff
@@ -89,6 +92,11 @@ export type TablePaginatorProps = {
      * @default (page: number) => `Go to page ${page}`
      */
     goToPageLabel?: (page: number) => string;
+    /**
+     * A label shown next to the page size selector. Use `null` to hide.
+     * @default 'Rows per page'
+     */
+    rowsPerPageLabel?: string | null;
```

Comment:
How come this label doesn't change the visual test images?

Reply:
It's because the table is not wide enough in the test environment.